.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - checkout
       - check_changes
       - restore_cache:
-          key: test-install-{{ .Branch }}
+          key: test-install-{{ .Branch }}-{{ checksum "setup.py" }}-{{ checksum "environment.yml" }}
```

Comment:
These changes are not needed, see also discussion elsewhere

Reply:
meh, I you're kinda right, I think they shouldn't be needed here, but the Core issue that Stef has a PR for, I think that's prob good to have, anyway - I'll remove them from here :+1:

Reply:
Thanks!